### DISCUSSION TITLE
Security audit of LdapRecord (result pdf)

### DISCUSSION THREAD

**Discussion by NicolasCARPi** [DirectoryTree/LdapRecord#725] [Show and tell]
Hello,

[Cure53](https://cure53.de/), a pentest company, performed an audit of LdapRecord library while working on Passbolt integration, which uses this lib.

@stevebauman congratulations, the feedback is very good! :clap: :tada: 

Here is the report: https://help.passbolt.com/assets/files/PBL-09-report.pdf

Given that the report is public, I thought it would be beneficial to all users to know about it. @stevebauman did you know about this?

**Comment by stevebauman** [comment]
Thanks @NicolasCARPi! I did get contacted about the pentest report a while ago, but I didn't post the results (I wasn't sure at the time if they wanted to keep it semi-private or not). 

I agree that this should be posted on the docs so it gives developers additional confidence in the library! 👍

**Reply by NicolasCARPi** [reply]
Yeah, I suggest adding a section in the README or wherever to mention it, with a link to the report (you might want to host it yourself though).
